### ISSUE TITLE
Explain how to concat-fields / extract more than one field as key/value in a variable

### ISSUE THREAD

**Issue by domdorn** [marcusolsson/grafana-json-datasource#89]
I have a json structure like this 
```
{
 "items": [ 
  { "id": 5, "name": "Vienna" }, 
  { "id": 6, "name": "Salzburg"}
 
 ]
 ..
 }
 ```
 
I try to create a variable "countyId" which allows the user to select the county "Vienna" or "Salzburg" and use the id (5 or 6) in follow up queries. 

I'm trying since more than three hours various ways of specifying my field, but I'm unable to come up with a way to get them used together, or concat them into a string so I can split them up in a regex. 

These are a few of the patterns I have tried, but to no avail: 
```
$.items[*][id,name] // this gives me 5, Vienna, 6, Salzburg
$.items[*].id // this gives me just the id, which is not user friendly
$.items[*].name // this gives me the name, which would be user friendly, but i cannot use it to resolve other parameters
$.concat($.items[*].name,$.items[*].id) // does not work, although I hoped it would ( https://stackoverflow.com/questions/45941225/how-to-concatenate-json-elements-with-jsonpath-jayway ) 
$.items[*] | { .name, .id } | @concat
$.items[*].name@string() + 'aaa'
```

so I need a way to either use two fields of an object for text + value or concat two fields into a string, so I can separate them using regex.

Please advice.

**Comment by marcusolsson** [comment]
This sounds like a duplicate of https://github.com/marcusolsson/grafana-json-datasource/issues/79? Could you check it out to see if that issue would fit your use case?

**Comment by marcusolsson** [comment]
I've just published [v1.1.0](https://grafana.com/grafana/plugins/marcusolsson-json-datasource/) to the marketplace, which includes experimental support for this. Would you mind checking it out and letting me know whether it addresses your use case?

**Comment by marcusolsson** [comment]
Closing due to inactivity.
